autogen.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+autoreconf --install
+automake --add-missing --copy > /dev/null 2>&1
```

Comment:
Autoreconf will run automake for you if you add `AM_INIT_AUTOMAKE` to configure.ac. It will do the copying of missing files, because you passed --install flag.

But I have noticed that you don't use any of automake features nor have any `.am` files. Just remove this line and add the install-sh script to git.